_people/abdullah_ghani.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "Abdullah Ghani "
+collection: people
+permalink: /people/abdullah-ghani
+position: ms
```

Comment:
I think there is a position `undergrad`, unless you are starting a master with us, and that would mean that your title needs to change to Master student

Reply:
I would say so and then we change it to `ms`. Thank you!